README.md

```diff
@@ -211,6 +212,14 @@ The fail comment condition is generated with [Lodash template](https://lodash.co
 
 > check the [GitLab API Issue object](https://docs.gitlab.com/ee/api/issues.html#single-issue) for properties which can be used for the filter
 
+#### publishToCatalog
+
+**Note**: This is an EXPERIMENTAL option that might change in the future.
```

Comment:
if considering this experimental, is it worth publishing as an alpha or beta release to get feedback from interested folks before releasing in a stable version?

Reply:
Yep, good point. I have retargeted this one to `next`.

Reply:
Based on the feedback from https://github.com/semantic-release/gitlab/pull/843 I have retargeted this to a fresh `beta` branch.